rest-api/cron.yaml

```diff
@@ -4,6 +4,11 @@ cron:
   schedule: every day 04:00
   timezone: America/New_York
   target: offline
+- description: Daily public metrics
+  url: /offline/PublicMetricsRecalculate
+  schedule: every day 03:30
```

Comment:
You might actually want to push this back a bit... BiobankSamplesImport will influence metrics on who is a full participant, and may not be done by the time this starts.

Reply:
Done